packages/server/src/plugin/landingPage/default/types.ts

```diff
@@ -22,6 +22,16 @@ export interface ApolloServerPluginLandingPageDefaultBaseOptions {
    * variables loaded in the UI.
    */
   variables?: Record<string, any>;
+
+  /**
+   * The ID of a collection, paired with an operation ID to populate in the Sandbox on load. 
+   *
+   * You can find these values from a registered graph in Studio by
+   * clicking the ... menu next to an operation in the Explorer of that graph and
+   * selecting View operation details.
+   */
+  collectionId?: string;
+  operationId?: string;
```

Comment:
We don't expect one without the other here, right? I think I'd suggest something like:
```ts
defaultOperation?: {
  collectionId: string;
  operationId: string;
}
```
That way we can give them a TS error for one without the other. (or you could get funky with the typings to keep them top level, but I think I like the nesting and the naming)

Another idea that comes to mind that we probably don't want but hey, just in case. A single string "ref" like: `collectionId@operationId` which wouldn't feel too unfamiliar because graph refs.

Reply:
I wrapped! Just noting that I added this to the shared types, but its only supported in the embeds, I just moved it out of this shared base type into the specific embed: true types 👍🏻